### ISSUE TITLE
Monitor keywords/games

### ISSUE THREAD

**Issue by camjac251** [arye321/Twitch-Stream-Recorder#1]
Would it be possible to have the script monitor a game's page and/or keywords in the title?

I'd like to record all streams of one game without having to manually startup multiple scripts for individual channels

**Comment by arye321** [comment]
Yes I can make that. Do you want to monitor the title or game name or both?

**Comment by camjac251** [comment]
Well it would record any channels live in a game category, but for those people who don't appear in the game directory then it could monitor the titles for certain keywords related to the game name. like abbreviations.

**Comment by arye321** [comment]
oh I miss understood your first post. what you're asking might take a lot more work. I'll try working on it this week, no promises.

**Comment by camjac251** [comment]
I asked another github user who had a similar python script as yours. They had some useful information about it [https://github.com/Instinctlol/automatic-twitch-recorder/issues/1](https://github.com/Instinctlol/automatic-twitch-recorder/issues/1)

**Comment by camjac251** [comment]
Did you see that link?
